go.mod

```diff
@@ -34,6 +34,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0
 	github.com/otiai10/copy v1.14.0
+	github.com/pkg/errors v0.9.1
```

Comment:
You should use `errors` and `fmt.Errorf` instead of the deprecated package `github.com/pkg/errors`

Reply:
ok, done.
@glours